scripts/build/update_version_stamp.lua

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env lua
+--
+-- Helper script to update Git version stamp.
```

Comment:
Every script "helps" , so "helper" qualifier is always noise.

Reply:
But some are meant to be run directly, while others are not..?

Reply:
? It's a script that takes parameters. No one will be confused.

Reply:
It is just similar to putting it in "builds" vs having it in "scripts" directly.. there are scripts that are meant to be run by users directly, and others which are run from the build.

Reply:
It's "premature explanation" / organization. We are nowhere close to having so many scripts to justify more hierarchy.